src/aircrafts/C310/fcs.jl

```diff
@@ -1,3 +1,4 @@
+using FlightMechanics
```

Comment:
I think that, as this files (`C310/aerodynamics.jl`, `C310/propulsion.jl`, `C310/fcs.jl` `C310/C310.jl` ...)  are included in `Aircrafts.jl` these imports are not necessary. I don't know if: it is a good practice to include them in every included file, or if it is better to move them to `Aircrafts.jl`.

Any clue?

Reply:
i dont think it will work that way. i mean we are importing Aircrafts.jl in FlightMechanics.jl. so if we import FlightMechanics into Aircrafts.jl doesnt it create problem?. by doing that we are importing Aircrafts.jl into Aircrafts.jl itself.

Reply:
we are just using some functions in this file like DEG2RAD and IN2M. so if import like FlightMechanics: DEG2RAD. would this have an effect on coverage. 
we are now only importing functions we required so does this reduce the  decrease in covarage??

Reply:
> i dont think it will work that way. i mean we are importing Aircrafts.jl in FlightMechanics.jl. so if we import FlightMechanics into Aircrafts.jl doesnt it create problem?. by doing that we are importing Aircrafts.jl into Aircrafts.jl itself.

I think I didn't explain this well. I was thinking about the fact that in `Aircraft.jl` I have `include` to `C310/aerodynamics.jl`, `C310/propulsion.jl`, `C310/fcs.jl` `C310/C310.jl` `F16/...` ... so once the code is injected in `Aircraft.jl` the `using FlightMechanics` is repeated many times. To my understanding, this is not a problem and the import will only be executed once, but I was wondering if other packages normally place multiple `using` (in every included file) or just one at the top level file. 

I tried to get a clarification on a previous stackoverflow question https://stackoverflow.com/questions/51819338/julia-split-module-into-multiple-files/51819722#51819722 

In my opinion, this is just highlighting that the organization is "too pythonic" and maybe julia tends to larger source files. I will think about joining, at least,  all the different files for the same aircraft. Of coure in a different PR. What do you think?

I don't think we are importing Aircrafts.jl into Aircrafts.jl that way. It is just a matter of where to import `FlightMechanics`.

> we are just using some functions in this file like DEG2RAD and IN2M. so if import like FlightMechanics: DEG2RAD. would this have an effect on coverage.
> we are now only importing functions we required so does this reduce the decrease in covarage??

I don't think that has an effect on coverage, but I am not sure.

Reply:
you are right once we import FlightMechanics in top file we dont have to import it in other file. i will check this in local to see if it works correctly and then after that we will think about doing changes in master branch.

and as for coverage, i did little searching and it is affected by how many function is used in tests. so to improve it we need to have test for all the functions.